### PR TITLE
Notify dev-portal-working-group in slack on successful deployment

### PR DESCRIPTION
**Pull request by 10thfloor**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -104,3 +104,13 @@ jobs:
...
       - uses: superfly/flyctl-actions/setup-flyctl@master
       - run: flyctl deploy --config fly.production.toml -e GIT_SHA=$(git rev-parse HEAD)
+
+  notify: 
+    name: Notify Team
+    needs: [deploy-production]
+    uses: slackapi/slack-github-action@v1.22.0
+    with:
+      channel-id: 'C03D8F2T68H'
+      slack-message: "GitHub build result: ${{ job.status }}\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
```
Comment (10thfloor) [inline .github/workflows/deploy.yml:114]:
```suggestion
      slack-message: "Portal was deployed to production. 🚀 GitHub build result: ${{ job.status }}\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
```

**Review by schpet — APPROVED** [review]
👍🏻👍🏻👍🏻

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -104,3 +104,13 @@ jobs:
...
       - uses: superfly/flyctl-actions/setup-flyctl@master
       - run: flyctl deploy --config fly.production.toml -e GIT_SHA=$(git rev-parse HEAD)
+
+  notify: 
+    name: Notify Team
+    needs: [deploy-production]
+    uses: slackapi/slack-github-action@v1.22.0
+    with:
+      channel-id: 'C03D8F2T68H'
+      slack-message: "Portal was deployed to production. 🚀 GitHub build result: ${{ job.status }}\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
```
Comment (10thfloor) [inline .github/workflows/deploy.yml:114]:
```suggestion
      slack-message: "Portal was deployed to production. GitHub build result: ${{ job.status }}\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
```

**Review by 10thfloor — COMMENTED** [review]
Add staging and prod deploy notifications.

**Pull request merged**